MinesweeperBoard.py

```diff
@@ -1,24 +1,31 @@
 ''' classes to support playing a game of Minesweeper
 '''
 
+from blessed import Terminal
 from enum import Enum
 import random
 
 
+# instantiate just to get display 'constants'
+#   (this static-ness will only bite us if there's both
+#   interactive and non-interactive output)
+TERM = Terminal()
+
+
 class KnowledgeState(Enum):
     HIDDEN = 1
     FLAGGED = 2
     STEPPED = 3
 
     def __str__(self):
         if self == self.HIDDEN:
-            return '?'
+            return TERM.cyan('?')
         elif self == KnowledgeState.FLAGGED:
-            return '#'
+            return TERM.yellow('#')
         elif self == KnowledgeState.STEPPED:
-            return 'S'
+            return TERM.blue('S')
         else:
-            return 'E'
+            return TERM.red('E')
```

Comment:
These changes take away the invariant that each cell has length 1. That might cause us problems.